src/it/java/teammates/it/storage/sqlsearch/AccountRequestSearchIT.java

```diff
@@ -164,4 +165,26 @@ private static void verifySearchResults(List<AccountRequest> actual,
         AssertHelper.assertSameContentIgnoreOrder(Arrays.asList(expected), actual);
     }
 
+    @Test
+    public void testSqlInjectionSearchAccountRequestsInWholeSystem() throws Exception {
+        ______TS("SQL Injection test in searchAccountRequestsInWholeSystem");
+
+        if (!TestProperties.isSearchServiceActive()) { 
+            return; 
+        } 
```

Comment:
We should probably have the test segment divider after the check. If the search service is not active, we would not want it to display the string when the test is not going to execute anyway.

Also, we should not have any trailing spaces.

```suggestion
        if (!TestProperties.isSearchServiceActive()) {
            return;
        }

        ______TS("SQL Injection test in searchAccountRequestsInWholeSystem");
```